lib/rdkafka/consumer.rb

```diff
@@ -73,6 +71,7 @@ def pause(list)
       end
       tpl = list.to_native_tpl
       response = Rdkafka::Bindings.rd_kafka_pause_partitions(@native_kafka, tpl)
+
       if response != 0
         list = TopicPartitionList.from_native_tpl(tpl)
```

Comment:
here was a memory leak, it doesn't properly clean a pointer if response code is 0